docs/docs/api/CacheStore.md

```diff
@@ -14,7 +14,15 @@ The `MemoryCacheStore` stores the responses in-memory.
 **Options**
 
 - `maxEntries` - The maximum amount of responses to store. Default `Infinity`.
-- `maxEntrySize` - The maximum size in bytes that a response's body can be. If a response's body is greater than or equal to this, the response will not be cached.
+- `maxSize` - The maximum size in bytes that a response's body can be. If a response's body is greater than or equal to this, the response will not be cached.
+
+### `SqliteCacheStore`
+
+The `SqliteCacheStore` stores the responses in a SQLite database.
```

Comment:
Shall we add a note for the `node:sqlite` dependency in case someone attempts to use it within Node.js <= 22?